dds/DCPS/security/HandleRegistryImpl.cpp

```diff
@@ -0,0 +1,320 @@
+#include "dds/DCPS/security/HandleRegistryImpl.h"
+
+#include "dds/DCPS/GuidConverter.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace Security {
+
+HandleRegistryImpl::~HandleRegistryImpl()
+{
+  if (DCPS::security_debug.bookkeeping) {
+    ACE_DEBUG((LM_DEBUG, ACE_TEXT("(%P|%t) {bookkeeping} ")
+               ACE_TEXT("HandleRegistryImpl::~HandleRegistryImpl local datareader %d local datawriter %d remote participant %d remote datareader %d remote datawriter %d\n"),
+               local_datareader_crypto_handles_.size(),
+               local_datawriter_crypto_handles_.size(),
+               remote_participant_crypto_handles_.size(),
+               remote_datareader_crypto_handles_.size(),
+               remote_datawriter_crypto_handles_.size()));
```

Comment:
32/64-bit mismatch between log format string and parameter

Reply:
What's the correct way to do this?  Cast up to 64-bit int?

Reply:
There's a log format character for `size_t`